### ISSUE TITLE
4.26 (preview) package errors

### ISSUE THREAD

**Issue by Clockwork-Muse** [adamrehn/conan-ue4cli#9]
Calling `ue4 conan generate` on the 4.26 preview yields: 

```
Using user-specified engine root: /repositories/UnrealEngine
Removing the "ue4.26-Linux-x86_64-unknown-linux-gnu" Conan profile if it already exists...
Removing any previous versions of profile base packages...
Creating "ue4.26-Linux-x86_64-unknown-linux-gnu" Conan profile using autodetected settings...
Installing profile base packages...
Copying the "ue4.26-Linux-x86_64-unknown-linux-gnu" Conan profile into a new profile named "ue4"...
Retrieving thirdparty library list from UBT...
Removing any previous versions of generated wrapper packages for 4.26...
Generating and installing toolchain wrapper package...
  Wrapping clang: /repositories/UnrealEngine/Engine/Extras/ThirdPartyNotUE/SDKs/HostLinux/Linux_x64/v16_clang-9.0.1-centos7/x86_64-unknown-linux-gnu/bin/clang
  Wrapping lib++: /repositories/UnrealEngine/Engine/Source/ThirdParty/Linux/LibCxx/lib/Linux/x86_64-unknown-linux-gnu/libc++.a
Generating and installing wrapper package for APEX...
Traceback (most recent call last):
  File "/home/Clockwork-Muse/.pyenv/ue4cli/bin/ue4", line 8, in <module>
    sys.exit(main())
  File "/home/Clockwork-Muse/.pyenv/ue4cli/lib/python3.7/site-packages/ue4cli/cli.py", line 216, in main
    SUPPORTED_COMMANDS[command]['action'](manager, args)
  File "/home/Clockwork-Muse/.pyenv/ue4cli/lib/python3.7/site-packages/conan_ue4cli/main.py", line 48, in main
    SUBCOMMANDS[subcommand]['function'](manager, args[1:])
  File "/home/Clockwork-Muse/.pyenv/ue4cli/lib/python3.7/site-packages/conan_ue4cli/commands/generate.py", line 175, in generate
    PackageManagement.generateWrapper(lib, template, delegates, tempDir, channel, profile)
  File "/home/Clockwork-Muse/.pyenv/ue4cli/lib/python3.7/site-packages/conan_ue4cli/common/PackageManagement.py", line 26, in generateWrapper
    PackageManagement.install(packageDir, channel, profile)
  File "/home/Clockwork-Muse/.pyenv/ue4cli/lib/python3.7/site-packages/conan_ue4cli/common/PackageManagement.py", line 16, in install
    return Utility.run(['conan', 'create', '.', 'adamrehn/' + channel, '--profile=' + profile] + args, cwd=packageDir)
  File "/home/Clockwork-Muse/.pyenv/ue4cli/lib/python3.7/site-packages/conan_ue4cli/common/Utility.py", line 31, in run
    '\nstdout: "{}"\nstderr: "{}"'.format(stdout, stderr)
Exception: child process ['conan', 'create', '.', 'adamrehn/4.26', '--profile=ue4.26-Linux-x86_64-unknown-linux-gnu'] failed with exit code 1
stdout: "Exporting package recipe
APEX/ue4@adamrehn/4.26: A new conanfile.py version was exported
APEX/ue4@adamrehn/4.26: Folder: /home/Clockwork-Muse/.conan/data/APEX/ue4/adamrehn/4.26/export
APEX/ue4@adamrehn/4.26: Exported revision: 8580ad497621d925fe8a1d4b58479b55
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=clang
compiler.libcxx=libc++
compiler.version=9
os=Linux
os_build=Linux
[options]
[build_requires]
*: toolchain-wrapper/ue4@adamrehn/4.26
[env]
UNREAL_ENGINE_VERSION=4.26
APEX/ue4@adamrehn/4.26: Forced build from source
Installing package: APEX/ue4@adamrehn/4.26
Requirements
    APEX/ue4@adamrehn/4.26 from local cache - Cache
    ue4lib/ue4@adamrehn/profile from local cache - Cache
Packages
    APEX/ue4@adamrehn/4.26:089c4f7a625e6973490fd9baeee0990e0ea3a28e - Build
    ue4lib/ue4@adamrehn/profile:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
Build requirements
    toolchain-wrapper/ue4@adamrehn/4.26 from local cache - Cache
Build requirements packages
    toolchain-wrapper/ue4@adamrehn/4.26:8010bbd5cef041065a4ea6a72a9c2962222d70ed - Cache

Installing (downloading, building) binaries...
toolchain-wrapper/ue4@adamrehn/4.26: Already installed!
ue4lib/ue4@adamrehn/profile: Already installed!
APEX/ue4@adamrehn/4.26: Applying build-requirement: toolchain-wrapper/ue4@adamrehn/4.26
APEX/ue4@adamrehn/4.26: Configuring sources in /home/Clockwork-Muse/.conan/data/APEX/ue4/adamrehn/4.26/source
APEX/ue4@adamrehn/4.26: Copying sources to build folder
APEX/ue4@adamrehn/4.26: Building your package in /home/Clockwork-Muse/.conan/data/APEX/ue4/adamrehn/4.26/build/089c4f7a625e6973490fd9baeee0990e0ea3a28e
APEX/ue4@adamrehn/4.26: Generator txt created conanbuildinfo.txt
APEX/ue4@adamrehn/4.26: Calling build()
APEX/ue4@adamrehn/4.26: Package '089c4f7a625e6973490fd9baeee0990e0ea3a28e' built
APEX/ue4@adamrehn/4.26: Build folder /home/Clockwork-Muse/.conan/data/APEX/ue4/adamrehn/4.26/build/089c4f7a625e6973490fd9baeee0990e0ea3a28e
APEX/ue4@adamrehn/4.26: Generated conaninfo.txt
APEX/ue4@adamrehn/4.26: Generated conanbuildinfo.txt
APEX/ue4@adamrehn/4.26: Generating the package
APEX/ue4@adamrehn/4.26: Package folder /home/Clockwork-Muse/.conan/data/APEX/ue4/adamrehn/4.26/package/089c4f7a625e6973490fd9baeee0990e0ea3a28e
APEX/ue4@adamrehn/4.26: Calling package()
"
stderr: "APEX/ue4@adamrehn/4.26: WARN: This conanfile has no build step
Using user-specified engine root: /repositories/UnrealEngine
ERROR: APEX/ue4@adamrehn/4.26: Error in package() method, line 54
        details = UE4Lib("APEX")
        KeyError: 'PublicAdditionalLibraries'
"
```

**Comment by meetakshay99** [comment]
I too am facing the same issue. Did you find a fix for this?

**Comment by adamrehn** [comment]
Thanks for reporting this, I'll investigate and see what changes need to be made to fix compatibility with Unreal Engine 4.26.0.

**Comment by adamrehn** [comment]
Just fixed the underlying error in ue4cli (<https://github.com/adamrehn/ue4cli/commit/6e0d256b295f07ce4327cc48cde0a2eaefc518ed>) and bumped the required version of ue4cli in 2c27341. Fix is live in ue4cli version 0.0.49 and conan-ue4cli version 0.0.35.
